backend/api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('', views.main, name = "Kiosk"),
+    path('training/', views.training, name = "Training Model"),
```

Comment:
This will be done internally. I had the training of the model in a different URL just for the purpose of testing out whether the functions I made in Django worked properly and communicated with the 'faux' frontend setup.